qa/setup_packages.py

```diff
@@ -62,67 +69,79 @@ def get_pyvers_name(name, cuda):
              pass
     return ""
 
+def get_version(package_version_entry):
+    if isinstance(package_version_entry, tuple):
```

Comment:
Just an idea. Maybe instead of tuple we should create some simple class to keep that data.

Reply:
I think it's an overkill (for now). If I third argument goes into place I might consider it